components/admin_console/cluster_settings.jsx

```diff
@@ -117,7 +117,7 @@ export default class ClusterSettings extends AdminSettings {
         }
 
         return (
-            <SettingsGroup>
+            <SettingsGroup container={true}>
```

Comment:
Since we're setting `container={true}` most of the times that we use `SettingsGroup`, we could make that the default value so that we have to set it to false in the few cases that we don't need it.

Reply:
Makes sense.